test/e2e/framework/service/jig.go

```diff
@@ -378,12 +384,12 @@ func (j *TestJig) SanityCheckService(svc *v1.Service, svcType v1.ServiceType) {
 		if svc.Spec.ExternalName != "" {
 			framework.Failf("unexpected Spec.ExternalName (%s) for service, expected empty", svc.Spec.ExternalName)
 		}
-		if svc.Spec.ClusterIP != api.ClusterIPNone && svc.Spec.ClusterIP == "" {
```

Comment:
It's redundant; if `ClusterIP == ""` then necessarily `ClusterIP != None`.

(The original author presumably wrote it that way because they were thinking "if ClusterIP isn't None, then make sure the service got assigned a ClusterIP".)